### DISCUSSION TITLE
React Native support with AWS Chime Messaging

### DISCUSSION THREAD

**Discussion by milan-techprescient** [aws/aws-sdk-js-v3#3969] [Q&A]
We have implemented aws chime message functionality in **React app (Web)** using [aws-sdk](https://www.npmjs.com/package/aws-sdk). It is working fine in web app.

Same as web app we have implemented **React Native app (Mobile)** using [@aws-sdk/client-chime](https://www.npmjs.com/package/@aws-sdk/client-chime). Now, the app builds successfully, but it is not loading & being stuck at splash screen only. Once we remove the library implementation code it works fine and loading successfully. Below is the code which is used to create the instance,
`const chimeClient = new AWS.Chime({
  region: REGION,
  credentials: {
    accessKeyId: AWS_ACCESS_KEY_ID,
    secretAccessKey: AWS_SECRET_ACCESS_KEY_ID,
  },
})`.

This behaviour is same for android and iOS app.

**Comment by yenfryherrerafeliz** [comment]
Hi @milan-techprescient, thanks for opening this discussion. Did you notice/check if is there any error log in the back?, if so, could you please post the error here?

Note: Please redact any sensitive information.

Thanks!

**Reply by milan-techprescient** [reply]
No, there is no such info or error is coming from SDK.
